_pages/openid_connect.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: OpenID Connect
+permalink: /openid-connect
```

Comment:
Actually, the way this Jekyll site is configured, the path is automatically equal to the file name, so you can remove `permalink` entirely (like the other pages).